### ISSUE TITLE
Rule proposal: `require-post-message-target-origin`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1307]
When calling [`window.postMessage()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) without `targetOrigin` argument, browser won't throw any error, but message can't receive by any window. I remember the first time I got into this, really drive me crazy.

## Fail

```js
foo.postMessage('hello!');
```

## Pass

```js
foo.postMessage('hello!', '*');
```
```js
foo.postMessage('hello!', 'http://example.com');
```

**Comment by sindresorhus** [comment]
This is now accepted.
